_includes/head.html

```diff
@@ -15,6 +15,7 @@
   <!-- Stylesheets -->
   <style>{%- include modern-normalize.css -%}</style>
   <style>{%- include inline-style.css -%}</style>
+  <style>{%- include inline-code-style.css -%}</style>
```

Comment:
I would probably load this with a link rel=stylesheet like the additional styles, it's not critical for layout and just adds a bit of colour, so I'd rather deprioritise it and not block parsing/rendering with it's inline download.

I guess `.highlight table td` and `.highlight table pre` would make sense to add to inline-styles though

Reply:
Now done \o/ I've just left it all as the rel=stylesheet, that css file's what `rougify style base16.monokai` spit out so I feel a lil icky messing with it manually.